src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx

```diff
@@ -107,4 +126,31 @@
   <data name="Downloading_IntelliSense_index_for_0" xml:space="preserve">
     <value>Downloading IntelliSense index for {0}</value>
   </data>
+  <data name="Regex_Comment" xml:space="preserve">
```

Comment:
Should all of these explain that the "Regex" part shouldn't be localized? /cc: @tmeschter

Reply:
Well... i'm not sure what the right thing is here.  If some language has an accepted localized term they use for 'Regular expression'... then i think it woul dbe ok to translate that...